### ISSUE TITLE
Propose directory structure and file naming convention for client-side js and css

### ISSUE THREAD

**Issue by amcoho** [smohadjer/formwizard#12]
We need guidelines as how to lay out the step or substep-specific js (validation, progressive enhancements) and css code on the disk and how to pull them into each step/substep. One idea is to keep form markup, js, and css together in one subfolder so adding or modify a step is easy and isolated from other steps.

**Comment by smohadjer** [comment]
Yes, we will keep styles and scripts for each view beside that view's markup. I will custom styles and js functionality to step two so we can show and test how this works.
